src/components/project_admin/suite_settings.vue

```diff
@@ -50,19 +48,50 @@
           </template>
         </select-object>
       </div>
-
       <div class="toggle-container form-field-wrapper">
-        <toggle v-model="d_suite.allow_network_access"
-                @input="$emit('field_change', d_suite)"
-                ref="allow_network_access">
+        <div>
+          <i class="fas fa-exclamation-triangle warning-tip"></i>
+          <tooltip width="large" placement="right">
+            {{WARNING_MESSAGE}}
+          </tooltip>
+        </div>
+        <toggle :value="d_suite.allow_network_access"
+                @input="on_toogle_allow_network"
+                ref="allow_network_access"
+                >
           <div slot="on">
             Allow network access
           </div>
           <div slot="off">
             Block network access
           </div>
         </toggle>
+        <modal v-if="d_show_allow_network_access_modal"
+           @close="d_show_allow_network_access_modal = false"
+           ref="allow_network_access_modal"
+           size="large"
+           :click_outside_to_close="true"
+           :show_closing_x="true">
+            <div class="modal-header">
+              Confirm Access
+            </div>
+            <div class="modal-body">
+              {{WARNING_MESSAGE}}
+            </div>
+            <div class="modal-button-footer">
+              <button data-testid="allow_network_access_button"
+                      class="red-button"
+                      :disabled="false"
+                      @click="allow_network">Allow
+              </button>
+              <button class="modal-cancel-button white-button"
+                      @click="d_show_allow_network_access_modal = false">
```

Comment:
Clicking "Cancel" currently doesn't reset the value of allow network access. Do you agree that it should?